packages/network/src/logic/tracker/Tracker.ts

```diff
@@ -138,12 +137,12 @@ export class Tracker extends EventEmitter {
         })
         this.extraMetadatas[source] = extra
 
-        const spidKey = SPID.toKey(stream.id, stream.partition)
+        const streamPartId = toStreamPartID(streamPart.id, streamPart.partition)
 
         // update topology
-        this.createTopology(spidKey)
-        this.updateNodeOnStream(source, stream)
-        this.formAndSendInstructions(source, spidKey)
+        this.createTopology(streamPartId)
+        this.updateNodeOnStream(source, streamPart)
```

Comment:
I think the 2nd argument may be wrong here? Wonder what happened.

Reply:
It's correct, but I'll improve the readability